Import/Esri/ArcGISRuntime/Toolkit/Controls/QmlApi/TimeSliderController.qml

```diff
@@ -18,26 +18,84 @@ import QtQuick 2.4
 import Esri.ArcGISRuntime 100.3
 
 /*!
-    \qmltype TimeSlider
+    \qmltype TimeSliderController
     \ingroup ArcGISQtToolkit
     \ingroup ArcGISQtToolkitQmlApi
     \inqmlmodule Esri.ArcGISRuntime.Toolkit.Controls
     \since Esri.ArcGISRutime 100.3
-    \brief
+    \brief The controller for the Time Slider tool.
+
+    The controller presents the temporal range of the data as a number of steps.
+    These steps allow the temporal extent to be set and animated by stepping through
+    the range.
 */
 Item {
     id: controller
 
+    /*!
+      \qmlproperty GeoView geoView
+      \brief the GeoView for this tool to \a geoView.
+
+      The view should be either a MapView or a SceneView.
+      */
     property GeoView geoView
+
+    /*!
+     \internal
+     */
     property bool isMapView: geoView && geoView.objectType === "MapView"
+
+    /*!
+     \internal
+     */
     property var operationalLayers: geoView ? (isMapView ? geoView.map.operationalLayers : geoView.scene.operationalLayers) : null
+
+    /*!
+     \internal
+     */
     property var timeAwareLayers: []
+
+    /*!
+     \qmlproperty int numberOfSteps
+     \brief The total number of required steps to cover the full time extent.
+
+     This figure is based on the full temporal range of the data in the geoView
+     and the time intervals used by the data.
+     */
     property int numberOfSteps: -1
+
+    /*!
+     \internal
+     */
     property TimeValue timeStepInterval
+
+    /*!
+     \internal
+     */
     property int intervalMS: -1
+
+    /*!
+     \qmlproperty TimeExtent fullExtent
```

Comment:
are these properties in the controller mostly read only? Or does the developer need to set some of them to get the slider to work? If read only, we should doc